mockgcp/mockserviceusage/knownservices.go

```diff
@@ -17,6 +17,7 @@ package mockserviceusage
 var allServices = []string{
```

Comment:
Could we add a comment to explain how this list should be used?

Reply:
Added a comment